src/main/java/ch/ivyteam/ivy/maven/engine/OsgiRuntime.java

```diff
@@ -115,15 +115,19 @@ void runThreadWithProperties(Callable<?> function, int timeoutEngineStartInSecon
             "javax.annotation,ch.ivyteam.ivy.boot.osgi.win,ch.ivyteam.ivy.jaas," // original
             + "org.apache.log4j,org.apache.log4j.helpers,org.apache.log4j.spi,org.apache.log4j.xml," // add log4j
             + "org.slf4j.impl,org.slf4j,org.slf4j.helpers,org.slf4j.spi," // add slf4j
-            
-            + "javax.management,javax.management.openmbean,javax.xml.parsers," // since oxygen platform
-            + "sun.net.www.protocol.http.ntlm,com.sun.xml.internal.ws.util,com.sun.nio.zipfs,org.xml.sax," // since oxygen platform
-            + "org.w3c.dom," // since oxygen platform
-            + "javax.xml,javax.xml.datatype,javax.xml.namespace,javax.xml.transform,javax.xml.transform.dom,javax.xml.transform.sax,javax.xml.transform.stream,javax.xml.validation,javax.xml.xpath," // for java 11
+            + "javax.net.ssl," // validate openApi
```

Comment:
this one was missing: see 
https://jenkins.ivyteam.io/view/scrum-master/job/demo-projects/job/master/1178/console